### PR TITLE
test(tooling): resusciate integration tests

### PR DESCRIPTION
**Pull request by thescripted**
Get integration tests to work

[SPARK-312424](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-312424)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -81,10 +80,14 @@
...
   },
   "jest": {
+    "setupFiles": [
+      "dotenv/config"
+    ],
     "setupFilesAfterEnv": [
       "<rootDir>/jest.setup.js"
     ],
     "preset": "jest-puppeteer",
+    "testEnvironment": "node",
```
Comment (lalli-flores) [inline package.json:90]: What's the reason for making node the default instead of JSDOM?

File: src/PeopleSDKAdapter.integration.test.js [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
+ * @jest-environment jsdom
+ */
 import {from} from 'rxjs';
 import {delayWhen, skip} from 'rxjs/operators';
 
-import {createIntegrationTestUser, removeIntegrationTestUser, getPersonHydraID} from './testHelper';
-
 import WebexSDKAdapter from '.';
 
+import {
```
Comment (lalli-flores) [inline src/PeopleSDKAdapter.integration.test.js:9]: Should probably not change unrelated code

File: src/RoomsSDKAdapter.integration.test.js [hunk 3]
```diff
@@ -1,10 +1,19 @@
...
+ * @jest-environment jsdom
+ */
 import {from} from 'rxjs';
 import {delayWhen, skip} from 'rxjs/operators';
 
-import {createIntegrationTestUser, removeIntegrationTestUser} from './testHelper';
-
 import WebexSDKAdapter from '.';
 
+import {
```
Comment (lalli-flores) [inline src/RoomsSDKAdapter.integration.test.js:9]: Should probably not change unrelated code

File: src/PeopleSDKAdapter.integration.test.js [hunk 4]
```diff
@@ -1,10 +1,20 @@
...
+  createIntegrationTestUser,
+  removeIntegrationTestUser,
+  getPersonHydraID,
+} from './testHelper';
+
+// unmock `@webex/common` to avoid `@webex/common` being mocked
```
Comment (lalli-flores) [inline src/PeopleSDKAdapter.integration.test.js:15]:
```suggestion
// Jest auto-mocks by default, avoid `@webex/common` being mocked
```

File: package.json (hunk 1 above)
Comment (thescripted) [reply package.json:90]: It looks like a few browser-based queries fails when the testEnvironment is not set to jsdom. Things like `navigator.mediaDevices` appears undefined when running the unit tests. I'm not entirely sure why this is the case.

By default, the testEnvironment is set to node so this just makes the current test behavior explicit. It does make sense for jsdom to be the default although..

Comment (lalli-flores) [reply package.json:90]: The reason why I would lean towards making JSDOM the default environment for this repository is because the code for the adapter is meant to run on browsers

**Review by lalli-flores — APPROVED** [review]
